how-it-works/00_General/01_overview-of-the-internet-computer.card.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architecture of the Internet Computer
+---
+
+![](/img/how-it-works/subnet_architecture.png)
+
+# Architecture of the Internet Computer
+
+The Internet Computer implements the vision of a World Computer –  an open and secure blockchain-based network that can host all of humanity's computations in a secure and trustworthy way, witout needing to trust any single party to behave properly.
```

Comment:
Two things sound a bit odd to me:

- all of humanity's computations – that is, first of all, exaggerated (since we'll not do that anytime soon) and second it seems grammatically off.
- without needing to trust any single party – that sounds like one doesn't need to trust anyone, but one has to trust that 2/3 of the nodes are correct.